packages/dht/src/connection/WebSocket/WebSocketConnector.ts

```diff
@@ -0,0 +1,197 @@
+require('setimmediate')
+import { EventEmitter } from 'events'
+import {
+    IConnectionSource,
+    Event as ConnectionSourceEvent,
+    Event as ConnectionSourceEvents
+} from '../IConnectionSource'
+import { ClientWebSocket } from './ClientWebSocket'
+import { Event as ConnectionEvents, Event as ConnectionEvent, IConnection } from '../IConnection'
+import { ITransport } from '../../transport/ITransport'
+import { RoutingRpcCommunicator } from '../../transport/RoutingRpcCommunicator'
+import { RemoteWebSocketConnector } from './RemoteWebSocketConnector'
+import {
+    HandshakeMessage,
+    Message,
+    MessageType,
+    PeerDescriptor,
+    WebSocketConnectionRequest,
+    WebSocketConnectionResponse
+} from '../../proto/DhtRpc'
+import { WebSocketConnectorClient } from '../../proto/DhtRpc.client'
+import { DeferredConnection } from '../DeferredConnection'
+import { TODO } from '../../types'
+import { Logger } from '../../helpers/Logger'
+import { IWebSocketConnector } from '../../proto/DhtRpc.server'
+import { ServerCallContext } from '@protobuf-ts/runtime-rpc'
+import { v4 } from 'uuid'
+
+const logger = new Logger(module)
+
+export class WebSocketConnector extends EventEmitter implements IConnectionSource, IWebSocketConnector {
+    private WESOCKET_CONNECTOR_APP_ID = "websocketconnector"
```

Comment:
Are these `APP_ID` values kind of "reserved" for DHT's internal use? If that's the case, we could prefix the values e.g. with `SYSTEM/`

The `protocol` package uses similar notation for internal key exchange streams:
https://github.com/streamr-dev/network-monorepo/blob/main/packages/protocol/src/utils/KeyExchangeStreamID.ts

Reply:
added prefix "/system" to all service IDs